ui/src/components/yaml/styles.module.less

```diff
@@ -324,17 +324,14 @@
       }
     }
 
-    .yaml_content_diagnosisContent {
+    .yaml_content_diagnosisBody {
+      position: relative;
       width: 400px;
```

Comment:
The removed styles (`color`, `background`, `padding`, `font-size`, `line-height`) should be checked to ensure they are applied elsewhere to maintain consistent styling.